app/stylesheet/miq-structured-list.scss

```diff
@@ -263,7 +263,11 @@
     width: 100%;
   }
 
-  .miq-structured-list-accordion {
+  .miq-structured-list-accordion {      
+    .list_header {
```

Comment:
Hey, I have a doubt here. This would suit the summary with 2 columns. How about more than 3?

Reply:
I thought this summary page component only had 2 columns, do you know any examples of a 3 column one?

Reply:
can you check with the network security group - https://github.com/ManageIQ/manageiq-ui-classic/pull/7997

Reply:
The table looks the same still.

Before:
<img width="1413" alt="Screenshot 2024-07-15 at 9 23 20 AM" src="https://github.com/user-attachments/assets/e9552903-6928-4d77-9843-a51316c12dee">

After:
<img width="1411" alt="Screenshot 2024-07-15 at 9 26 24 AM" src="https://github.com/user-attachments/assets/1e74e7f3-3fee-46ef-83d5-f290dc9f5e67">